README.md

```diff
@@ -40,12 +40,6 @@ yarn lint
 
 [ESLint](https://eslint.org) check. _Recommended to run before commit_.
 
-```bash
-yarn flow
```

Comment:
As Sasha said, this command replaced on `npx tsc --noEmit`, for types checking. Also it included in precommit hook